src/components/Icon/IconDashboard.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import cx from 'classnames';
+import PropTypes from 'prop-types';
+import StyleObjectPropType from '../../prop-types/style';
+import './Icon.scss';
+
+let lastInstanceId = 0;
+
+const propTypes = {
+    className: PropTypes.string,
+    style: StyleObjectPropType,
+};
+
+const defaultProps = {
+    className: null,
+    style: null,
+};
+
+const IconDashboard = ({
+    className,
+    style,
+}) => {
+    lastInstanceId += 1;
+    const iconTitleId = `icon-dashboard-title-${lastInstanceId}`;
+
+    return (
+        <svg
+            className={cx('uir-icon uir-icon-dashboard', className)}
+            style={style}
+            viewBox="0 0 32 24"
+            aria-labelledby={iconTitleId}
+        >
+            <title id={iconTitleId}>Dashboard Icon</title>
+            <g stroke="none" strokeWidth="1" fill="none" fillRule="evenodd">
+                <g fillRule="nonzero">
+                    <path className="uir-icon-fill-charcoal" fillOpacity="0.5" transform="translate(0,0)" d="M8 1.155L1.206 5.077v7.846L8 16.845l6.794-3.922V5.077L8 1.155zM8 0l7.794 4.5v9L8 18 .206 13.5v-9L8 0z" />
+                    <path className="uir-icon-fill-charcoal" fillOpacity="1" transform="translate(7.8,4.5)" d="M8 1.155L1.206 5.077v7.846L8 16.845l6.794-3.922V5.077L8 1.155zM8 0l7.794 4.5v9L8 18 .206 13.5v-9L8 0z" />
+                    <path className="uir-icon-fill-charcoal" fillOpacity="0.2" transform="translate(15.46,0)" d="M8 1.155L1.206 5.077v7.846L8 16.845l6.794-3.922V5.077L8 1.155zM8 0l7.794 4.5v9L8 18 .206 13.5v-9L8 0z" />
```

Comment:
Do we need this `uir-icon-fill-charcoal` class? It's not defined anywhere.
Where possible, classes on icon paths should describe the shapes to make it easier to style individual paths. See for example IconMore here: https://github.com/Mudano/ui-react/blob/master/src/components/Icon/IconMore.js#L34-L36
where classes are "uir-icon-more-oval-1"
If the colour changes, "-fill-charcoal" won't make any sense.

Btw, those icons aren't visible when I run the code locally.

Reply:
Pull the latest – it looks like when I pulled this code into it's own branch I lost the icon.scss file.

We use the same fill and stroke colour across multiple icons so I felt it made more sense to share those classes (see icon.scss)

Reply:
Well, I wholeheartedly disagree with having colour names in classes. The classes are supposed to help people style the icon, including a change of colour. So they should semantically describe the element, not how it looks. Because the look might change.

I don't want to block you so I can approve this, though.
But since you're changing existing class names, this will have to be a breaking change.